dart/lib/src/hub.dart

```diff
@@ -73,6 +77,7 @@ class Hub {
         try {
           sentryId = await item.client.captureEvent(
             event,
+            stackTrace: stackTrace,
```

Comment:
What about we make the stackTrace part of the hint?
The idea of hint was to have a single .. hint to deal with these extra things without having to expand the public API

Reply:
the `hint` still could be used for other stuff as we do on Android, but stackTrace on dart is captured separately so it makes sense IMO to have this expanded API.
on Java, a stack trace is always part of the exception, which is not this case.
The other way would be adding a new field to the event, which is confusing.